### DISCUSSION TITLE
Support loongarch and ppc64le

### DISCUSSION THREAD

**Discussion by liuzhilin12** [linuxdeepin/developer-center#3471] [Features Request & Ideas | 特性请求 & 头脑风暴]
I want to support loongarch and ppc64le.

**Comment by BLumia** [comment]
Hi, you're welcome to contribute to the kernel source that deepin maintained. The source is located at https://github.com/deepin-community/kernel, and I encourage you join the [kernel SIG](https://github.com/deepin-community/SIG/tree/master/sig/deepin-kernel) so it could be easier to work with kernel issues :)
